topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.md

```diff
@@ -0,0 +1,176 @@
+key_points:
+- Several MD engines are available in BRIDGE
+- Workflows are available for common simulations tasks such as equilibration and production dynamics for various ensembles (NVT, NPT)
+- You've run an equilibrium and production MD simulation using NAMD
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: computational-chemistry
+    tutorials:
+      - analysis-md-simulations
+contributors:
+- chrisbarnettster
+- tsenapathi
+- simonbray
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+In this tutorial we will perform a simulation with the popular [NAMD](http://www.ks.uiuc.edu/Research/namd/) molecular dynamics software. Please note NAMD tools are not currently available on a public Galaxy server due to licensing issues. If you are interested in following this tutorial, you will need to download the [BRIDGE docker container](https://github.com/scientificomputing/BRIDGE) and [download NAMD](https://www.ks.uiuc.edu/Development/Download/download.cgi?PackageName=NAMD) yourself.
+
+This tutorial is made up of two parts. In the first section, we will look at preparation of a system (solvation, charge neutralisation, energy minimisation) using CHARMM. In the second section, we will perform an equilibration and production simulation, using NAMD. If you already completed the [Setting up molecular systems]({{ site.baseurl }}{% link topics/computational-chemistry/tutorials/setting-up-molecular-systems/tutorial.md %}) tutorial, which covers the use of the CHARMM graphical user interface (GUI), you have already prepared your system, so go straight to the [second section](#md-simulations-with-namd), using the files you prepared earlier.
+
+The process can be accomplished by selecting each tool from the tools menu, or by importing the workflow. The workflow method is most efficient and the individual tools used in the workflows are discussed below. The entire workflow (preparation + simulation) is shown below for [CHARMM and NAMD](#workflows).
+
+
+
+> ### {% icon details %} NVT, NPT and statistical mechanics theory
+>
+> [See Statistical Mechanics, McQuarrie for more in depth theory ISBN:9781891389153](https://books.google.co.za/books/about/Statistical_Mechanics.html?id=itcpPnDnJM0C&redir_esc=y)
+>
+{: .details}
+
+# System preparation with CHARMM
+
+If you already prepared your system using the CHARMM-GUI, and saved the output files, you can skip this section.
+
+## Setup
+
+Initially, we need to prepare a protein-ligand system in CHARMM. 
+
+This tool will:
+- solvate the protein-ligand complex, using the TIP3P water model
+- neutralise the system, using 0.05M NaCl
+- conduct a short energy minimisation 
+
+> ### {% icon hands_on %} Hands-on: initial processing
+>
+> Run **System Setup** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"psf input"*: Protein structure file of the protein-ligand system. (Input dataset)
```

Comment:
Do you need first to upload the data? If so, could you add an upload hands-on box here? thanks

Reply:
@chrisbarnettster, could you look at this? Thanks for the suggestions @bebatut, I think I've fixed the other things.

Reply:
added Hands-on Data upload.